bgpd/bgp_evpn_private.h

```diff
@@ -30,8 +30,9 @@
 
 #define RT_ADDRSTRLEN 28
 
-/* EVPN prefix lengths. This reprsent the sizeof struct prefix_evpn  */
-#define EVPN_ROUTE_PREFIXLEN     224
+/* EVPN prefix lengths. This represents the sizeof struct evpn_addr
+ * in bits  */
+#define EVPN_ROUTE_PREFIXLEN (sizeof(struct evpn_addr) * 8)
```

Comment:
Anuradha, 

The changes looks fine. However, I have a small question. 

what is the rational behind choosing the struct size as the PREFIX_LEN ? Is it actually the length of the key that FRR uses ? 

I checked the code and currently EVPN_ROUTE_PREFIXLEN is used to assign to prefix_len if struct prefix_evpn.

Just want to understand more on the usage and rational behind choosing this size.

Reply:
@srimohans 

The `route_table` code (lib/table.h) is structured to store prefixes and compares only (prefix length) number of bits;  in this case because it is storing EVPN addrs, we must specify the length of the EVPN address so it compares the entire address.  Otherwise it'd try to compare only part of it (or nothing.)